buildSrc/src/main/java/org/elasticsearch/gradle/ElasticsearchDistribution.java

```diff
@@ -182,22 +184,30 @@ public void setFlavor(Flavor flavor) {
         this.flavor.set(flavor);
     }
 
-    public boolean getBundledJdk() {
+    public boolean hasBundledJdk() {
```

Comment:
I think one thing for not originally naming this "has" is that we configure one of these when we _request_ a distribution and so the naming there makes a bit less sense when the context is that we are _asking_ for a distribution w/ or w/o a bundled JDK.

Reply:
I've reverted these renames.